lib/bindings/bindings.cc

```diff
@@ -207,12 +205,11 @@ BindingInstance::bind_value(const char *name, int value)
     lua_getglobal(this->lua, start);
     isnil = lua_isnil(this->lua, -1);
     lua_pop(this->lua, 1);
+    Debug("lua", "isnil? %s", isnil ? "yes" : "no");
```

Comment:
We don't need the local `isnil` variable any more. You can remove it and the associated code.